sphinx/config.py

```diff
@@ -51,6 +51,8 @@ class Config(object):
         # the real default is locale-dependent
         today_fmt = (None, 'env', [str]),
 
+        figure_language = (None, 'env', [str]),
+        figure_language_filename = ('%(root)s.%(language)s%(ext)s', 'env', [str]),
```

Comment:
Ok. I read the `Config.check_types` and I understand type hinting is no effect.
But, it is still **worth** to me. The config variable declarations having type hinting gives type-information **intentionally** to code readers. (maybe it is useful for document generation :-)
If none, of course it works fine, but code readers might not notice the type-information. because it is implicit.

Again, I think explicit type hinting is better than implicit one even if there is no effect.
What do you think?
If we disagree about this, I will remove these declarations.